.devcontainer/setup.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+set -eux
+
+# Install volta
+curl https://get.volta.sh | bash -s -- --version latest
+export VOLTA_HOME="${HOME}/.volta"
+export PATH="${VOLTA_HOME}/bin:${PATH}"
+volta install node
+volta install yarn
```

Comment:
The idea here is that we can test theme frontend stuff using multiple versions of node. Again, this is intended to be reused at some point.